beacon-chain/core/execution/upgrade_test.go

```diff
@@ -1,9 +1,10 @@
-package execution
+package execution_test
```

Comment:
Not sure why but I got `/root/.cache/bazel/_bazel_root/5c04516d923b21f838ef18b7f8ed7f88/sandbox/processwrapper-sandbox/4627/execroot/prysm/beacon-chain/core/execution/upgrade_test.go: import of "github.com/prysmaticlabs/prysm/testing/util"`. Must be some circular dependency issue